dataflow/src/main/java/org/checkerframework/dataflow/cfg/CFGBuilder.java

```diff
@@ -1339,10 +1353,11 @@ public void setSuccessor(BlockImpl successor) {
                         }
 
                         // exceptional edges
-                        for (Entry<TypeMirror, Set<Label>> entry : en.getExceptions().entrySet()) {
-                            TypeMirror cause = entry.getKey();
-                            for (Label label : entry.getValue()) {
-                                Integer target = bindings.get(label);
+                        for (Entry<TypeMirror, Set<Pair<TypeMirror, Label>>> entry :
+                                en.getExceptions().entrySet()) {
+                            for (Pair<TypeMirror, Label> label : entry.getValue()) {
```

Comment:
Throughout, you should chose new variable names if their type changed. `label` is not the right name for a `Pair<TypeMirror, Label>`; find a suitable name and use it throughout.